website/src/utils/Utils.tsx

```diff
@@ -103,4 +109,236 @@ export const closeTables = 70;
+          new: data.result.errors.new,
+          p: data.result.errors.p,
+          delta: data.result.errors.delta,
+          insignificant: data.result.errors.insignificant,
+        },
+        totalComponentsCpuTime: {
+          title: "Total CPU / Query",
+          old: data.result.total_components_cpu_time.old,
+          new: data.result.total_components_cpu_time.new,
+          p: data.result.total_components_cpu_time.p,
+          delta: data.result.total_components_cpu_time.delta,
+          insignificant: data.result.total_components_cpu_time.insignificant,
+        },
+        vtgateCpuTime: {
+          title: "vtgate",
+          old: data.result.components_cpu_time.vtgate.old,
+          new: data.result.components_cpu_time.vtgate.new,
+          p: data.result.components_cpu_time.vtgate.p,
+          delta: data.result.components_cpu_time.vtgate.delta,
+          insignificant: data.result.components_cpu_time.vtgate.insignificant,
+        },
+        vttabletCpuTime: {
+          title: "vttablet",
+          old: data.result.components_cpu_time.vttablet.old,
+          new: data.result.components_cpu_time.vttablet.new,
+          p: data.result.components_cpu_time.vttablet.p,
+          delta: data.result.components_cpu_time.vttablet.delta,
+          insignificant: data.result.components_cpu_time.vttablet.insignificant,
+        },
+        totalComponentsMemStatsAllocBytes: {
+          title: "Total Allocated / Query",
+          old: data.result.total_components_mem_stats_alloc_bytes.old,
+          new: data.result.total_components_mem_stats_alloc_bytes.new,
+          p: data.result.total_components_mem_stats_alloc_bytes.p,
+          delta: data.result.total_components_mem_stats_alloc_bytes.delta,
+          insignificant:
+            data.result.total_components_mem_stats_alloc_bytes.insignificant,
+        },
+        vtgateMemStatsAllocBytes: {
+          title: "vtgate",
+          old: data.result.components_mem_stats_alloc_bytes.vtgate.old,
+          new: data.result.components_mem_stats_alloc_bytes.vtgate.new,
+          p: data.result.components_mem_stats_alloc_bytes.vtgate.p,
+          delta: data.result.components_mem_stats_alloc_bytes.vtgate.delta,
+          insignificant:
+            data.result.components_mem_stats_alloc_bytes.vtgate.insignificant,
+        },
+        vttabletMemStatsAllocBytes: {
+          title: "vttablet",
+          old: data.result.components_mem_stats_alloc_bytes.vttablet.old,
+          new: data.result.components_mem_stats_alloc_bytes.vttablet.new,
+          p: data.result.components_mem_stats_alloc_bytes.vttablet.p,
+          delta: data.result.components_mem_stats_alloc_bytes.vttablet.delta,
+          insignificant:
+            data.result.components_mem_stats_alloc_bytes.vttablet.insignificant,
+        },
+      };
+    }) || []
+  );
+}
+
+export function formatCompareResult(
+  data: CompareResult
+): MacroBenchmarkTableData {
+  return {
```

Comment:
I think `formatCompareData` and this, should be simplified in the future to avoid duplicating code.